tests/test_pinblock.py

```diff
@@ -2,6 +2,9 @@
 from psec import pinblock
 
 
+from random import randbytes, randint, choice
```

Comment:
Probably the __init__.py has to be removed or adapted to make this import work in this environment. 
Btw, on my local machine I have pytest installed globally not in the virtual machine and it works.

Reply:
Try testing it like this from inside the project directory.

```
python -m pytest
```

When pytest is run as a module it adds current folder to path. And that makes everything discoverable.